src/olympia/abuse/migrations/0038_auto_20240819_1343.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.2.15 on 2024-08-19 13:43
+
+from django.db import migrations, models
+
+from olympia.constants.abuse import DECISION_ACTIONS
+
+
+def migrate_escalate_action_to_is_forwarded(apps, schema_editor):
+    CinderDecision = apps.get_model('abuse', 'CinderDecision')
+    CinderJob = apps.get_model('abuse', 'CinderJob')
+
+    CinderJob.objects.filter(
+        decision__action=DECISION_ACTIONS.AMO_ESCALATE_ADDON
+    ).update(
+        is_forwarded=True,
```

Comment:
That migration should also set `resolvable_in_reviewer_tools=True` right ?

Reply:
they should already have that - it would have been set when the `AMO_ESCALATE_ADDON` decision came in.

Reply:
Ah, right. I was testing with a manually created decision and I had set a bunch of fields but forgot to update the `CinderJob` I was using.